### PR TITLE
Add CLI guide for creating a node pool

### PR DESCRIPTION
**Pull request by kuosandys**
Towards [giantswarm/giantswarm#18343](giantswarm/giantswarm#18343).

This PR adds a CLI guide to the Cluster Details > Worker Nodes tab > Add Node Pool form, and provides information on how to template and create a node pool using the Management API.

The contents of the node pool manifest aligns with user input from the Add Node Pool form.

Collapsed:
Screen Shot 2021-08-19 at 10 23 18

Expanded:
Screen Shot 2021-08-19 at 16 38 26
Screen Shot 2021-08-19 at 16 38 46

### CODE REVIEW

**Comment by marians** [comment]
Looking good already!

Regarding the behaviour: The general idea is that we only show required flags. If a flag sets a value just like the default would, then we should omit it.

Example: When nodes count min=3 is the default, we should not have the flag. Currently the flag only disappears when min is 0. Same for max. Our default is 10 so the flag should not be present when max=10, but it should be present if max is set to 0.

**Comment by kuosandys** [comment]
> Looking good already!
> 
> Regarding the behaviour: The general idea is that we only show required flags. If a flag sets a value just like the default would, then we should omit it.
> 
> Example: When nodes count min=3 is the default, we should not have the flag. Currently the flag only disappears when min is 0. Same for max. Our default is 10 so the flag should not be present when max=10, but it should be present if max is set to 0.

@marians I've made the changes and updated the screenshots - the flags should only be displayed if they differ from the default values

**Comment by marians** [comment]
Availability zones in form and CLI guide don't match:

image

image

**Comment by marians** [comment]
Wait, my bad. That's actually working as expected.

**Comment by marians** [comment]
image

The `--context gs-ghost` flag should ideally be in the 2nd position, right after `kubectl`.

**Review by marians — APPROVED** [review]
Awesome!

File: src/components/MAPI/workernodes/guides/CreateNodePoolGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+              nodesMin,
+              nodesMax,
+              output: `cluster-${clusterName}-nodepool.yaml`,
+            }),
+            withFormatting()
+          )}
+        />
+        <CLIGuideStep
+          title='3. Apply the manifest'
+          command={`kubectl --context ${context} apply -f cluster-${clusterName}-nodepool.yaml`}
```
Comment (kuosandys) [inline src/components/MAPI/workernodes/guides/CreateNodePoolGuide.tsx:98]: Updated `context` flag's position 👍

**Pull request merged**